src/OpenTelemetry.Instrumentation.AspNetCore/OpenTelemetry.Instrumentation.AspNetCore.csproj

```diff
@@ -13,8 +13,23 @@
 
   <ItemGroup>
     <ProjectReference Include="$(RepoRoot)\src\OpenTelemetry\OpenTelemetry.csproj" />
-    <PackageReference Include="Microsoft.AspNetCore.Http.Abstractions" Version="$(MicrosoftAspNetCoreHttpAbstractionsPkgVer)" />
     <PackageReference Include="Microsoft.AspNetCore.Http.Features" Version="$(MicrosoftAspNetCoreHttpFeaturesPkgVer)" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
```

Comment:
its not clear why do we have to add netcoreapp3.1 and net5.0 targets, as they are already covered by netstandard2.1 target.
For netstandard2.1, we can take frameworkreference, and for everything else, we can packagereference Microsoft.AspNetCore.Http.Abstractions

Reply:
I am afraid that it is not possible.

When I change the `ItemGroup` for .NetStandard2.1 like this:
```csproj
<ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.1'">
  <FrameworkReference Include="Microsoft.AspNetCore.App" />
</ItemGroup>
```

`dotnet build` returns following error:
```log
dotnet/sdk/5.0.201/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.Sdk.FrameworkReferenceResolution.targets(326,5): error NETSDK1073: The FrameworkReference 'Microsoft.AspNetCore.App' was not recognized [../opentelemetry-dotnet/src/OpenTelemetry.Instrumentation.AspNetCore/OpenTelemetry.Instrumentation.AspNetCore.csproj]
```